packages/lesswrong/lib/collections/comments/schema.ts

```diff
@@ -634,6 +633,22 @@ const schema: SchemaType<DbComment> = {
     ...schemaDefaultValue(false),
   },
   
+  title: {
```

Comment:
I made this generic because I thought we might want to add the option for shortform in the near future